### PR TITLE
chore: Update docs manifest home page and icons

### PR DESCRIPTION
**Pull request by BrunoQuaresma**

### CODE REVIEW

**Review by kylecarbs — APPROVED** [review]
Seems fine, but there is this PR that fixed an issue: https://github.com/coder/coder/pull/1630

**Comment by BrunoQuaresma** [comment]
@kylecarbs good catch. cc.: @bpmct @jsjoeio we need to move the README to the docs to make it available on the docs website. Maybe what we can do is replicate the READMEs? Thoughts?

File: docs/manifest.json [hunk 1]
```diff
@@ -33,16 +43,11 @@
           "path": "./CODE_OF_CONDUCT.md"
         },
         {
-          "title": "Contributors",
-          "description": "See a contributors to Coder.",
-          "path": "./CONTRIBUTORS.md"
+          "title": "Security policy",
+          "description": "Learn how to report vulnerabilities in Coder.",
```
Comment (bpmct) [inline docs/manifest.json:47]:
```suggestion
          "description": "Our policy for security vulnerabilities in Coder.",
```

File: docs/manifest.json [hunk 2]
```diff
@@ -1,30 +1,40 @@
 {
   "versions": ["0.6.2"],
   "routes": [
+    {
+      "title": "Home",
+      "description": "Learn how to install and run Coder.",
```
Comment (bpmct) [inline docs/manifest.json:6]:
```suggestion
      "description": "Quickstart guide for using Coder.",
```

**Review by bpmct — APPROVED** [review]

File: docs/manifest.json (hunk 1 above)
Comment (khorne3) [reply docs/manifest.json:47]: Eh, this isn't really a policy -- maybe we just call it "Security" and leave it as "How to report vulnerabilities in Coder"

File: docs/manifest.json (hunk 2 above)
Comment (khorne3) [reply docs/manifest.json:6]: The README is more quickstart-ish, so I suggest "How to install and run Coder" as a simplification

**Review by khorne3 — APPROVED** [review]

**Comment by kylecarbs** [comment]
Should we make a separate README for the docs? I feel like they shouldn't necessarily be the same, since the routing might be a bit different.

**Comment by bpmct** [comment]
Agreed - I think the docs can have a separate readme. happy to help with this tomorrow. for now, we can just do the table of contents?

**Review by jsjoeio — COMMENTED** [review]
I think this looks great! I agree with changes suggested by @khorne3 and @bpmct so after that should be good to merge!

File: docs/manifest.json (hunk 1 above)
Comment (jsjoeio) [reply docs/manifest.json:47]: Fixed to Katie's point

File: docs/manifest.json (hunk 2 above)
Comment (jsjoeio) [reply docs/manifest.json:6]: Fixed

File: docs/manifest.json [hunk 3]
```diff
@@ -1,30 +1,46 @@
...
       "title": "Installation",
-      "description": "Learn how to install and deploy Coder.",
+      "description": "How to install and deploy Coder",
+      "icon": "<svg class=\"MuiSvgIcon-root jss172\" focusable=\"false\" viewBox=\"0 0 24 24\" aria-hidden=\"true\"><path d=\"M22.7 19l-9.1-9.1c.9-2.3.4-5-1.5-6.9-2-2-5-2.4-7.4-1.3L9 6 6 9 1.6 4.7C.4 7.1.9 10.1 2.9 12.1c1.9 1.9 4.6 2.4 6.9 1.5l9.1 9.1c.4.4 1 .4 1.4 0l2.3-2.3c.5-.4.5-1.1.1-1.4z\"></path></svg>",
       "path": "./install.md"
     },
+    {
+      "title": "Quickstart",
+      "description": "Create your first template and workspace",
+      "icon": "<svg class=\"MuiSvgIcon-root jss172\" focusable=\"false\" viewBox=\"0 0 24 24\" aria-hidden=\"true\"><path d=\"M22.7 19l-9.1-9.1c.9-2.3.4-5-1.5-6.9-2-2-5-2.4-7.4-1.3L9 6 6 9 1.6 4.7C.4 7.1.9 10.1 2.9 12.1c1.9 1.9 4.6 2.4 6.9 1.5l9.1 9.1c.4.4 1 .4 1.4 0l2.3-2.3c.5-.4.5-1.1.1-1.4z\"></path></svg>",
```
Comment (jsjoeio) [inline docs/manifest.json:25]: @BrunoQuaresma I didn't know what icon to use so I just copied another one. We can update before we make the site live

**Pull request merged**